src/media/css/app-list.styl

```diff
@@ -0,0 +1,72 @@
+/**
+ * App listing styles.
+ */
+
+@import 'lib';
+
+.app-list {
+    position: absolute;
+    top: $app-preview-height;
+    bottom: 0;
+    left: 0;
+
+    width: 100%;
+    overflow: auto;
+
+    ul {
+        padding: 4em 3em 3em 4em;
+    }
+
+    li {
+        float: left;
+        margin: 0 1em 1em 0;
+
+        list-style: none;
+    }
+}
+
+.app-list-app {
+    position: relative;
+
+    width: 150px;
+    height: 150px;
+    outline: 0;
+    border-radius: 100%;
+    overflow: hidden;
+
+    color: $greyscale-light-grey;
+    background-color: $greyscale-light-grey;
+
+    backface-visibility: hidden;
+
+    transition: border-width .5s,
+                transform .5s;
+
+    &:focus {
+        z-index: 1;
+
+        border: 8px solid $greyscale-white;
+
+        transform: scale(1.5);
+    }
+
+    img {
+        width: 100%;
+        height: auto;
+    }
+
+    .name {
+        position: absolute;
+        bottom: 0;
+        left: 0;
```

Comment:
Yea not an issue in the title but will be an issue in the preview area. If RTL support is needed you may want to follow the pattern we have in fireplace:

`.html-ltr {float: right;margin-left: 20px;}`
`.html-rtl {float: left;margin-right: 20px;}`

Flexbox is even better since it handles RTL for you in cases of floating though I am not sure about the level of flexbox support in the version of gecko that shipped for TVs.